### ISSUE TITLE
Exceptionally higher number of ASVs in some samples than others after pooling

### ISSUE THREAD

**Issue by yanxianl** [benjjneb/dada2#936]
Hi！

In a recent study, we profiled the response of fish gut microbiota to a dietary intervention. Besides sampling the luminal contents and mucosal tissue from the proximal and distal intestine, we also collected feed and water samples. We're interested in estimating species richness in the samples so we used `pooling` for the sample inference. When comparing the alpha-diversity before and after `pooling`, we found exceptionally higher number of ASVs in the feed samples than others after pooling. As shown below, the differences were big enough to change the conclusion regarding the species riechness in the feeds compared to other types of samples. 
![independent_pooling_contrast](https://user-images.githubusercontent.com/25475156/73929603-67819180-48d5-11ea-8b45-40c573b6828f.jpg)

When plotting the correlation between the number of ASVs before and after sample `pooling`, we found that the number of ASVs were ~3 times higher for feed samples after pooling.
![independent_pooling_cor](https://user-images.githubusercontent.com/25475156/73928229-08228200-48d3-11ea-915a-34ae2bafb358.jpg)

We're not sure whether this is due to an exceptionally higher number of rare ASVs or a much higher sequencing error rate in the feed samples. Could you help us figure out which results we should trust?

Here are the DADA2 denoising reports. Data were generated from 2 Miseq runs.
[dada2 report.zip](https://github.com/benjjneb/dada2/files/4164564/dada2.report.zip)

-Yanxian

**Comment by benjjneb** [comment]
Hm, that is interesting...

My first guess is that there might be a systematic difference between the bacterial concentration in the feeds samples, in particular that they are much lower in bacterial biomass. This would mean a larger fraction of contaminants (which can include cross-contamination) would be present in those samples, and pooling is very good at picking up low abundance contaminants (and chimeras).

Does that seem possible? I'm not sure how to directly test that, but it might be at least worth looking at the taxonomies of the additional ASVs detected in the feed samples when pooling, versus those detected when not pooling.

Given it's a different sample type, there is the possibility that feed samples have a higher fraction of rare taxa for some reason too.

**Comment by yanxianl** [comment]
Hi! 

> My first guess is that there might be a systematic difference between the bacterial concentration in the feeds samples, in particular that they are much lower in bacterial biomass.

Indeed, we expected the microibial DNA to be much lower in the feeds. Suprisingly, we found that feed samples carry considerable amount of bacterial DNA that is comparable to intestinal contents. 

> This would mean a larger fraction of contaminants (which can include cross-contamination) would be present in those samples

The majority of reads in the negative controls were filtered during sequence denoising by DADA2. The `observed ASVs ` I shown in the previous thread was calculated using the feature table with main contaminant ASVs removed. Hence, it's not contaminant taxa that are contributing to the higher `observed ASVs`. 

Cross-contamination is a good explanation of the exceptionally higher ASVs after sample pooling. I checked the sample layout in the plates during library preparation but I didn't find strong evidence of cross-contamination. 

> it might be at least worth looking at the taxonomies of the additional ASVs detected in the feed samples when pooling, versus those detected when not pooling.

Here's the shared ASVs found in the feeds using independent or pooled sample inference:
![feedShared](https://user-images.githubusercontent.com/25475156/74109830-421bae80-4b87-11ea-9d04-9633b46598f9.jpg)

Here's the additional ASVs found in the feeds when applying sample pooling:
![feedExtra](https://user-images.githubusercontent.com/25475156/74109859-8d35c180-4b87-11ea-830b-9118183cf7c5.jpg)

The majority of extra ASVs are assigned to bacteria taxa that are common in the fish gut, though there're several contaminant taxa not removed.

> Given it's a different sample type, there is the possibility that feed samples have a higher fraction of rare taxa for some reason too.

The number of sequences in the feed samples after denoising by DADA2 are 11171 and 18553 respectively, which are among the samples with least number of sequences. Moreover, the ratio of singletons in these 2 samples after sample pooling are 0.32 and 0.26, which are inverse to the number of sequences in the samples. Hence, I tend to believe feed samples actually possess a high microbial diverisity, though the possibity of cross-contamination can't be competely ruled out.

Thanks for your intepretations and help!
